base_layer/core/src/transactions/transaction_protocol/sender.rs

```diff
@@ -58,6 +58,8 @@ use tari_crypto::{
 
 /// This struct contains all the information that a transaction initiator (the sender) will manage throughout the
 /// Transaction construction process.
+// TODO: Investigate necessity to use the 'Serialize' and 'Deserialize' traits here; this could potentially leak
```

Comment:
This comment came about because a variable of type `UnblindedOutput` was added to this struct and it, in turn, imposed the necessity to have 'Serialize' and 'Deserialize' traits on `UnblindedOutput`, which is possibly the bigger problem.